lib/util.js

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+/**
+ * Retrieves the first set of capability invocation and delegation signers
+ * associated with the `didDocument` from the `keyPairs`.
+ *
+ * @param {object} options - The options to use.
+ * @param {string} options.didDocument - A DID Document containing
+ *   verification relationships for capability invocation and delegation.
+ * @param {string} options.keyPairs - A map containing keypairs indexed by
+ *   key ID.
+ *
+ * @returns {object} - A valid `invocationSigner` and `delegationSigner`
+ *   associated with the didDocument.
+ */
+export function getCapabilitySigners({didDocument, keyPairs}) {
+  const delegationSigner =
```

Comment:
We should throw an error if `capabilityDelegation` or `capabilityInvocation` are not available, or if `keyPairs` does not contain the VMs specified.

Reply:
Done in 84f5f2ba20ced7f885bdb53eb555b3bcbaa83fda.